packages/cli/test/commands/compile.test.js

```diff
@@ -36,4 +37,22 @@ describe('compile command', function() {
       });
     },
   );
+
+  itShouldParse(
+    'should call compile with typechain',
+    'compiler',
+    'zos compile --typechain=web3-v1 --typechain-outdir=foo --no-interactive',
```

Comment:
```suggestion
    'oz compile --typechain=web3-v1 --typechain-outdir=foo --no-interactive',
```
You can actually type anything instead of `zos` for example `sdfsdf` and it still will work. I think we should improve that.